### ISSUE TITLE
Optimization number of generated metrics

### ISSUE THREAD

**Issue by KVInventoR** [dmachard/dns-collector#25]
After a lot of my requests, now I see a big list of metrics, some of them can be combined with each other, example:

```
dnscollector_as_numbers_top_total{stream="alpha-vm01",number="15169"} 4259
dnscollector_as_owners_top_total{stream="alpha-vm01",owner="GOOGLE"} 4259

dnscollector_as_numbers_top_total{stream="alpha-vm01",number="14618"} 1222
dnscollector_as_owners_top_total{stream="alpha-vm01",owner="AMAZON-AES"} 1222
```
can be merged by labels

likes as:
dnscollector_as_stats{stream="alpha-vm01",as_number="14618",as_owner="GOOGLE"} 4259
dnscollector_as_stats{stream="alpha-vm01",as_number="14618",as_owner="AMAZON-AES"} 1222

**Comment by dmachard** [comment]
It's limited to top 100 items, I will try to merge it.

You can reduce the number of items to return in the config file:
```
subprocessors:
  statistics:
    top-max-items: 100
```

**Comment by dmachard** [comment]
new beta release with tag v0.14.0-b3 is available.

**Comment by KVInventoR** [comment]
@dmachard 
it's a little wrong:

```
dnscollector_as_stats_top_total{stream="beta-vm01",number="13188",owner="13188"} 6
dnscollector_as_stats_top_total{stream="beta-vm01",number="209366",owner="209366"} 5
dnscollector_as_stats_top_total{stream="beta-vm01",number="21497",owner="21497"} 5
dnscollector_as_stats_top_total{stream="beta-vm01",number="39729",owner="39729"} 5
dnscollector_as_stats_top_total{stream="beta-vm01",number="21334",owner="21334"} 5
```

**Comment by dmachard** [comment]
my mistake, a new beta release is available: v0.14.0-b4

**Comment by KVInventoR** [comment]
now, everything looks really good for me, thank you
